labelbox/schema/project.py

```diff
@@ -367,6 +367,34 @@ def create_prediction(self, label, data_row, prediction_model=None):
         res = self.client.execute(query_str, params)
         return Prediction(self.client, res["createPrediction"])
 
+    def enable_model_assisted_labeling(self, toggle:bool=True) -> bool:
+        """ Turns model assisted labeling either on or off based on input
+
+        Args:
+            toggle (Boolean): True or False boolean 
+        Returns:
+            True or False depending on the condition given
```

Comment:
would be more correct and understandable if you said here instead that it returns True if turned on and False if turned off